### ISSUE TITLE
Suggestions needed to improve model, trained on an indic language

### ISSUE THREAD

**Issue by SachinTelecmi** [SWivid/F5-TTS#940]
### Checks

- [x] This template is only for usage issues encountered.
- [x] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [x] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [x] I am using English to submit this issue to facilitate community communication.

### Environment Details

I am training a new indic language on single 4090 machine 
tokenizer pinyin

### Steps to Reproduce

![Image](https://github.com/user-attachments/assets/071407ef-fbbb-4572-8535-6f6ff1a17eba)
![Image](https://github.com/user-attachments/assets/45437137-fe34-43be-a203-ddde810990da)

### ❌ Actual Behavior

@SWivid the loss function is very oscillatory and non converging altho issue #838 says about non convergence and oscillation my issue is similar to like this but still its unclear to me after training for more than 100 epochs also why the result is poor is it ony depending on learning rate??can you help me with that i want to understand it!! To solve such issues what is needed and what are the main reasons behind it??

**Comment by SWivid** [comment]
Hi @SachinTelecmi , we saw some successful training on Indic. Maybe their config would help?
https://github.com/SWivid/F5-TTS/blob/main/src/f5_tts/infer/SHARED.md#hindi
https://github.com/AI4Bharat/IndicF5

**Comment by SachinTelecmi** [comment]
> Hi [@SachinTelecmi](https://github.com/SachinTelecmi) , we saw some successful training on Indic. Maybe their config would help? https://github.com/SWivid/F5-TTS/blob/main/src/f5_tts/infer/SHARED.md#hindi https://github.com/AI4Bharat/IndicF5

@SWivid thanks for the quick response glad for that!! one more thing  What you suggest seeing the loss curve can you tell me??  should i train more also lr should i change??

**Comment by SWivid** [comment]
we don't look the loss curve, lol; better infer and hear

**Comment by ShreyasPathe** [comment]
Hi! I have experience training speech models in Python and PyTorch and tuning convergence. I’d like to work on improving loss stability or convergence for the Indic language model. Could you assign this issue to me?
